e2e/features/case-progression.feature

```diff
@@ -82,10 +82,9 @@ Feature: Case progression
     When I request respondent evidence
     And I click the `Overview` tab
 
-    Then I should only see the `caseOfficer_awaitingRespondentEvidence_preUpload` case progress image
-
+    Then I should only see the `caseOfficer_awaitingRespondentEvidence` case progress image
```

Comment:
I think this should be _preUpload image isn't it? Looks like this is overriding to revert the latest changes. @bradsorour could confim this.